package.json

```diff
@@ -3,6 +3,16 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+<<<<<<< HEAD
```

Comment:
Michael -- looks like you have an unresolved merge conflict here. Make sure to get edit the package.json file to take out the stuff that shouldn't be there.